scripts/infra_k8s_check.sh

```diff
@@ -54,21 +54,10 @@ RESET_COLOR="$(tput sgr0 2>/dev/null || printf "\e[0m")"
 # Check things are installed
 ############################
 # Do not show the check again if it has already been installed
-ALREADY_INSTALLED_FILE="$HOME/.infra/installed"
+ALREADY_INSTALLED_FILE="$HOME/.aladdin/infra/installed"
```

Comment:
Does it matter that we won't find the 'installed' flag from a previous version?  And does anything remain in `~/.infra`?

Reply:
it just means that the infra check will run once after this lands, then the `installed` flag will be reset 👍 
`~/.infra` is no longer used